File: src/musikdroid/app/src/main/java/io/casey/musikcube/remote/util/Duration.java
package io.casey.musikcube.remote.util;

import java.util.Locale;

public class Duration {
    private Duration() {

    }

    public static String format(double seconds) {
        final int mins = ((int) seconds / 60);
        final int secs = (int) seconds - (mins * 60);
        return String.format(Locale.getDefault(), "%d:%02d", mins, secs);
    }
}

File: src/musikdroid/app/src/main/java/io/casey/musikcube/remote/util/Duration.kt
package io.casey.musikcube.remote.util

import java.util.*

object Duration {
    fun format(seconds: Double): String {
        val mins = seconds.toInt() / 60
        val secs = seconds.toInt() - mins * 60
        return String.format(Locale.getDefault(), "%d:%02d", mins, secs)
    }
}
